screens/ChatsScreen.js

```diff
@@ -27,55 +28,30 @@ import {
     Item,
     Input
 } from 'native-base';
-
-// import { WebBrowser } from 'expo';
-
-const chatNames = [
-    {
-        id: 1,
-        name: 'Best friends!'
-    },
-    {
-        id: 2,
-        name: 'Family ❤️'
-    },
-    {
-        id: 3,
-        name: 'OSU People'
-    }
-];
+import { fetchChats, setCurrentChat } from '../actions/chats';
 
 class ChatsScreen extends React.Component {
     static navigationOptions = {
         header: null
     };
 
-    createMessage = () => (this.props.createMessage(1, 1, "Test", "David", { David: ':joy:', Nick: ':laughing:' }));
-    setCurrentUser = () => (this.props.setCurrentUser("David"));
-    setCurrentChat = () => (this.props.setCurrentChat(1));
-    createChat1 = () => (this.props.createChat(
-        1,
-        'Tea Chats',
-        ['David', 'Nick']
-    ));
-    createChat2 = () => (this.props.createChat(
-        2,
-        'Coffee Chats',
-        ['David', 'Ryan']
-    ));
+    componentDidMount() {
+        this.props.fetchChats();
```

Comment:
if you need to fetch data when something loads, do it in componentDidMount().
note - we can also add loading states later when fetching or sending data.